plugins/source/stripe/codegen/recipes/account.go

```diff
@@ -0,0 +1,14 @@
+package recipes
+
+import (
+	"github.com/stripe/stripe-go/v74"
+)
+
+func init() {
```

Comment:
The use of `init` to append to a global `AllResources` is quite... creative? 😅 I don't know how I feel about it, but I do admit that it at least avoids needing to remember to add each resource to a slice in `codegen/main.go`, so maybe it's better than what we have. Possibly something to debate a bit, because I think we should stick to one convention or the other, at least